public/documents/fari-rpgs/charge-srd.md

```diff
@@ -372,6 +372,21 @@ Once you've decided how many charges you want to consume, roll and interpret the
 - On a `6`, the `consequence` is avoided.
 - On a `66`, the `consequence` is avoided, and you get +1 position for your next roll.
 
+> #### Designer Note: Hacking Determination Rolls
+> 
+> Determination rolls act as a mechanism for the players to say "no, I don't think so" when something bad happens to their character.
+> 
+> There are two important things you should look at if you want to hack this mechanic. 
```

Comment:
```suggestion
> There are two important things you should look at if you want to hack this mechanic:
```

Reply:
You think I should place a dot here instead of the colon? I used a colon because of the list belie

Reply:
It's a sentence to introduce the list below, so it sounds a good idea for me